trySwift/SpeakerDetailViewController.swift

```diff
@@ -11,7 +11,7 @@ import TrySwiftData
 
 class SpeakerDetailViewController: UITableViewController {
 
-    var speaker: Speaker!
```

Comment:
... and there isn't really another way of doing this in that case. In case of a speaker not being passed in, the view would just be empty.